### ISSUE TITLE
imgur embedding not working.

### ISSUE THREAD

**Issue by heycitizen** [kasperisager/vanilla-oembed#4]
Hi,

It seems that imgur embedding is not working on my forum.
I am running vanilla 2.2 and oembed version 2.0.1 
I was previously using the Mediator plugin

The test gallery I use is: http://imgur.com/gallery/qO7EU

When I post this I get a blank line as seen here:
http://i.imgur.com/BW4S8OF.png
When inspecting the html for the post this ls what i see:

```
<div class="Message">
Testing imgur oembed:<br>
<br>
<a href="http://imgur.com/gallery/qO7EU" target="_blank" rel="nofollow"></a><br>
</div>
```

I do not get any console errors. oembed works for other services (flickr, TED etc)

What is wrong and how do i fix it? I have tried both firefox and chrome with no extensions.

cheers.

**Comment by heycitizen** [comment]
I have solved this by adding a file _rich.php_ in the views folder containing:

```php
<?php echo $media->html; ?>
```

**Comment by kasperisager** [comment]
Solved in bd8152f2919c53faba3bb22db861804fb73fc5c2
